### PR TITLE
BigQuery authentication sample.

### PR DESCRIPTION
**Pull request by tswast**
Lists datasets on a project using cURL.

### CODE REVIEW

**Comment by tswast** [comment]
This sample will be used in an explanation of how the requests to BigQuery are authorized with OAuth2.

File: bigquery/authorization/snippets.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+set -e
+
+PROJECT_ID="$1"
+if [[ -z "$PROJECT_ID" ]] ; then
+  echo "Usage: $0 project-id"
+  exit 1
+fi
+
+# [START get_token]
+ACCESS_TOKEN="$(gcloud auth print-access-token)"
```
Comment (theacodes) [inline bigquery/authorization/snippets.sh:25]: I *think* gcloud is strongly recommending against using print-access-token.

Comment (tswast) [reply bigquery/authorization/snippets.sh:25]: Do you have an alternative?

**Comment by theacodes** [comment]
Not a good one.

**Pull request merged**